web/explorer/src/components/NavBar.vue

```diff
@@ -13,8 +15,9 @@ import { RouterLink } from "vue-router";
         <template #end>
             <div class="flex align-items-center gap-3">
                 <a
+                    v-if="!isBundle"
                     v-ripple
-                    v-tooltip.right="'Download capa Explorer Web for offline usage'"
+                    v-tooltip.bottom="'Download capa Explorer Web for offline usage'"
```

Comment:
what's the cases we want to use consistently?
all lowercase? `capa explorer web` or as is?

Reply:
I intend to tackle this as part of the next PR. As you stated, rename everything to `capa explorer web`